src/frontend/magnify/src/components/design-system/Card/MagnifyCard.test.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import MagnifyCard from './MagnifyCard';
+
+describe('MagnifyCard', () => {
+  it('should render successfully a MagnifyCard', async () => {
+    render(
+      <MagnifyCard title={'Test'}>
+        <div>Test body</div>
+      </MagnifyCard>,
+    );
+    const header = screen.getByRole('heading', { level: 3 });
+    expect(header).toBeInTheDocument();
+    expect(header).toHaveTextContent('Test');
+    expect(screen.getByText('Test body')).toBeInTheDocument();
```

Comment:
By using `getBy` methods, if no element is found an error is raised so the test will fail.

https://testing-library.com/docs/queries/about/#types-of-queries
```suggestion
    screen.getByText('Test body');
```